src/samplebufferutil.h

```diff
@@ -0,0 +1,17 @@
+#ifndef SAMPLEBUFFERUTIL_H
+#define SAMPLEBUFFERUTIL_H
+
+#include "samplebuffer.h"
+#include "sampleutil.h"
+
+// Fills the whole buffer with zeroes
+inline void clear(SampleBuffer* pBuffer) {
```

Comment:
I'd rather not have these in the global namespace. Why not attach them to the SampleBuffer object as you had them?